db/orders.js

```diff
@@ -0,0 +1,169 @@
+const {client} = require('./client');
+const { getProductById } = require('./products');
+
+async function createOrder({ status, userId }) {
+    const { rows: [order] } = await client.query(`
+        INSERT INTO orders(status, "userId")
+        VALUES ($1, $2)
+        RETURNING *;
+    `, [status, userId]);
+
+    return order;
+};
+
+async function addProductsToOrder(order){
+    try {
+        order.product = await getProductById(order.productId);
```

Comment:
There should be multiple products on one order.  This should instead be a function called `getProductsByOrderId` or something, and it shoudl query the db for products that, when joined with `order_products`, find the products that have an `order_products.orderId` that matches the one passed in.  BUT I do like the fact that it's a helper function that will be used in multiple other functions.

Reply:
This will break the front-end very easily, so must be fixed!  Also, please do create the Express route AND frontend for this, so that you can test it, and make sure the data is usable in the format you need.